internal/provider/provider.go

```diff
@@ -214,6 +214,7 @@ func New(version string) func() *schema.Provider {
 				"cloudflare_custom_hostname":                        resourceCloudflareCustomHostname(),
 				"cloudflare_custom_pages":                           resourceCloudflareCustomPages(),
 				"cloudflare_custom_ssl":                             resourceCloudflareCustomSsl(),
+				"cloudflare_device_policy":                          resourceCloudflareDeviceSettingsPolicy(),
```

Comment:
the resource name here is `cloudflare_device_policy` but everywhere we're referring to it as device settings policy. which do you think is best fitting so we can align on consistent naming?

Reply:
Good point, I think `cloudflare_device_settings_policy` fits better

Reply:
Did I not generate the docs correctly? Trying to update docs with `cloudflare_device_settings_policy` but re-running `make docs` undoes your commits

Reply:
I'll take a look now. it's probably a naming thing